.github/workflows/integration_tests_reusable.yml

```diff
@@ -86,13 +86,16 @@ jobs:
       fail-fast: false
       matrix:
         group: ${{ fromJSON(needs.generate-matrices.outputs.e2e) }}
+        # Empty value uses default
+        react: ['', '18.3.1']
     uses: ./.github/workflows/build_reusable.yml
     with:
       afterBuild: |
         # e2e and ${{ inputs.test_type }} tests with `node run-tests.js`
 
         export NEXT_TEST_CONTINUE_ON_ERROR=TRUE
         export NEXT_E2E_TEST_TIMEOUT=240000
+        export NEXT_TEST_REACT_VERSION=${{ matrix.react }}
```

Comment:
Is Bash fine if this is empty e.g. `export NEXT_TEST_REACT_VERSION=`? We use `NEXT_TEST_REACT_VERSION="${{ matrix.react }}"` in the other places currently.

Reply:
I also wondered about this, but this should work if the following also works, right?

```sh
export NEXT_TEST_MODE=${{
  inputs.test_type == 'development' && 'dev' || 'start'
}}
```

Reply:
🤞